### ISSUE TITLE
No emoji support

### ISSUE THREAD

**Issue by weq** [EvotecIT/PSTeams#4]
It would seem that the module doesn't allow for posting of emojis. If this is a limitation on the Teams backend API or the module itself I haven't investigated.

`Send-TeamsMessage -uri $uri -MessageTitle "Foo" -MessageText "Foo ❌ bar" `
It seems that the module interprets this using the -Color option somehow if it is because of the console breaking it up or anything I don't know either, but it returns all the -Color options.

**Comment by PrzemyslawKlys** [comment]
Hi,

Thanks for reporting this. I've fixed problem with -Color and I've fixed problem with MessageText. As for emojis it doesn't seem to work. But it may be related to encoding maybe?

Try Update-Module PSTeams and retry please

Przemek

**Comment by PrzemyslawKlys** [comment]
Fix provided in #8 added possibility for this:

![image](https://user-images.githubusercontent.com/15063294/55550039-98060700-56d7-11e9-9514-8e9c6aaaadb4.png)

```
Send-TeamsMessage -uri $TeamsID -MessageTitle "Foo" -MessageText "Foo ❌ 😀 💩 bar" -Verbose
```

Should we do celebrate dance? :-) I'll publish this soon

**Comment by PrzemyslawKlys** [comment]
https://github.com/EvotecIT/PSTeams/blob/master/Examples/Example4.ps1

Please keep in mind you may need UTF-8 with BOM file encoding.
